src/mapper/value-mapper.factory.ts

```diff
@@ -39,6 +46,12 @@ export const valueMapperFactory = async () => {
         return Array.isArray(value) ? value : []
       case TYPES.ContentType:
         return contentType(string(value))
+      case TYPES.ExtractedDomains:
+        if (isExtractDomains) {
+          return extractDomains(string(value), extractDomainsWhitelist)
+        } else {
+          return undefined
```

Comment:
If domains extraction is disabled, just return undefined like if it was private.